demo/src/App.js

```diff
@@ -21,20 +24,33 @@ export default class App extends PureComponent {
     Object.keys(source.channels).forEach((channel, i) => {
       sliderValues[channel] = [0, 20000];
       colorValues[channel] = colorOptions[i];
+      channelsOn[channel] = true;
       sliders[channel] = withStyles({
         root: {
           color: `rgb(${colorOptions[i]})`
         }
       })(Slider);
+      checkboxes[channel] = withStyles({
+        root: {
+          color: `rgb(${colorOptions[i]})`,
+          '&$checked': {
+            color: colorOptions[i]
+          }
+        },
+        checked: {}
+        // eslint-disable-next-line react/jsx-props-no-spreading
+      })(checkBoxProps => <Checkbox color="default" {...checkBoxProps} />);
```

Comment:
It's kind of confusing with the ignore syntactically separated from the line it applies to. Maybe something like.

```suggestion
      })(checkBoxProps => (
        // eslint-disable-next-line react/jsx-props-no-spreading
        <Checkbox color="default" {...checkBoxProps} />)
      );
```